### DISCUSSION TITLE
GitHub secrets not found: "Input required and not supplied"

### DISCUSSION THREAD

**Discussion by Xunnamius** [community/community#24924] [Actions]
<p>I’m experiencing something similar to the issue noted here: <a href="https://github.community/t/github-secrets-not-be-found/18007" class="inline-onebox">Github Secrets not be found</a></p>
<p>Around a week ago, secrets stopped being passed into workflow runs triggered by dependabot. Seems like they are only being passed into runs triggered specifically by my account or other collaborators. It’s strange, because 1) nothing has changed with these workflow files in weeks and 2) its occurring across all my repositories that use secrets during workflow runs triggered by dependabot pushing to a branch.</p>
<p>More details here: <a href="https://github.com/actions/github-script/issues/122" class="inline-onebox" rel="noopener nofollow ugc">Error: Input required and not supplied: github-token · Issue #122 · actions/github-script · GitHub</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That is an intentional security fix to how pull requests from Dependabot are treated:</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/" target="_blank" rel="noopener">The GitHub Blog</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2019/03/blog-card.png?fit=5001%2C2626" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/" target="_blank" rel="noopener">GitHub Actions: Workflows triggered by Dependabot PRs will run with read-only...</a></h3>

<p>GitHub Actions: Workflows triggered by Dependabot PRs will run with read-only permissions</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Xunnamius** [comment]
<p>Wow, didn’t notice this. Thanks!</p>
